doc/source/ray-air/examples/serving_guide.ipynb

```diff
@@ -0,0 +1,658 @@
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "local_checkpoint = Checkpoint.from_dict({\"increment\": 2})\n",
+    "local_predictor = AdderPredictor.from_checkpoint(local_checkpoint)\n",
+    "assert local_predictor.predict(np.array([40])) == np.array([42])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It worked! Now let's serve it behind HTTP. In Ray Serve, the core unit of HTTP service is called a [`Deployment`](serve-key-concepts-deployment). It turns a Python class into queryable HTTP endpoint. For Ray AIR, Serve provides a `ModelWrapperDeployment` make it simpler. You don't need to implement any Python classes. You just pass in your predictor and checkpoint instead.\n",
+    "\n",
+    "The deployment takes several arguments. It requires two arguments to start:\n",
+    "- `predictor_cls (Type[Predictor] | str)`: The predictor Python class. Typically you just need to use the builtin integration from Ray AIR like `TorchPredictor`. Alternatively, you can specify the class path to import such predictor like `\"ray.ml.integrations.torch.TorchPredictor\"`.\n",
+    "- `checkpoint (Checkpoint | str)`: A checkpoint instance, or uri to load checkpoint from.\n",
+    "\n",
+    "The following cell showcase how to create a deployment with our `AdderPredictor`\n",
+    "\n",
+    "For more about Ray Serve the framework, checkout [its documentation](rayserve)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-05-20 11:30:34,597\tINFO services.py:1483 -- View the Ray dashboard at \u001b[1m\u001b[32mhttp://127.0.0.1:8265\u001b[39m\u001b[22m\n",
+      "\u001b[2m\u001b[36m(ServeController pid=66733)\u001b[0m INFO 2022-05-20 11:30:40,278 controller 66733 checkpoint_path.py:17 - Using RayInternalKVStore for controller checkpoint and recovery.\n",
+      "\u001b[2m\u001b[36m(ServeController pid=66733)\u001b[0m INFO 2022-05-20 11:30:40,386 controller 66733 http_state.py:115 - Starting HTTP proxy with name 'SERVE_CONTROLLER_ACTOR:cjsRQe:SERVE_PROXY_ACTOR-node:127.0.0.1-0' on node 'node:127.0.0.1-0' listening on '127.0.0.1:8000'\n",
+      "\u001b[2m\u001b[36m(HTTPProxyActor pid=66737)\u001b[0m INFO:     Started server process [66737]\n",
+      "\u001b[2m\u001b[36m(ServeController pid=66733)\u001b[0m INFO 2022-05-20 11:30:42,814 controller 66733 deployment_state.py:1217 - Adding 1 replicas to deployment 'Adder'.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from ray import serve\n",
+    "from ray.serve.model_wrappers import ModelWrapperDeployment\n",
+    "\n",
+    "# Create Ray Serve instance\n",
+    "serve.start()\n",
+    "\n",
+    "# Deploy the model behind HTTP endpoint\n",
+    "ModelWrapperDeployment.options(name=\"Adder\").deploy(\n",
+    "    predictor_cls=AdderPredictor,\n",
+    "    checkpoint=local_checkpoint\n",
```

Comment:
I don't need to specify `json_to_ndarray` here?

Reply:
In the code snippet here https://ray--25003.org.readthedocs.build/en/25003/serve/http-adapters.html#ray-air-modelwrapper, we pass in `json_to_ndarray` to the `ModelWrapperDeployment`, so I'm a bit confused on when we need to do it or not.